### PR TITLE
Verify the comet impl as part of normal cli verification process

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: scripts/verify-comet.ts [hunk 1]
```diff
@@ -1,73 +0,0 @@
...
-}
-
-// We recommend this pattern to be able to use async/await everywhere
-// and properly handle errors.
-main()
-  .then(() => process.exit(0))
-  .catch((error) => {
-    console.error(error);
-    process.exit(1);
-  });
```
Comment (kevincheng96) [inline scripts/verify-comet.ts:73]: Unfortunately, we'll probably still need this script for when Comet is deployed not through the initial deploy process (e.g. governance proposals).

**Review by kevincheng96 — APPROVED** [review]
🙌

File: scripts/verify-comet.ts (hunk 1 above)
Comment (jflatow) [reply scripts/verify-comet.ts:73]: We can still run the deploy cli in those cases and it should work! (just ran it with simulate first to test, then for real, and the only action it does is attempt to verify the impl, which of course is already verified)

Comment (kevincheng96) [reply scripts/verify-comet.ts:73]: Oh nice! Should we document this in the README? Since it's less obvious how we can manually verify now.

Comment (kevincheng96) [reply scripts/verify-comet.ts:73]: Also probably good to just mention in general that we need to manually verify after every upgrade to Comet.

File: README.md [hunk 2]
```diff
@@ -218,10 +218,22 @@ These simulations are extremely useful for testing deployments before actually c
 
 0. Create the deployment script and configuration file, and test locally
 1. Open a PR containing the new deployment directory files
-2. Trigger the `run-deploy` workflow action through the GitHub UI
+2. Trigger the `deploy-market` workflow action through the GitHub UI
 3. Inspect the new `roots.json` which the workflow automatically commited to your PR
 4. Start using the new protocol deployment and/or create further migrations to modify it
 
+##### Verifying Deployments
```
Comment (kevincheng96) [inline README.md:225]: 👍

**Review by kevincheng96 — APPROVED** [review]

**Pull request merged**